apps/site/next-data/generators/blogData.mjs

```diff
@@ -58,55 +64,66 @@ const getFrontMatter = (filename, source) => {
  * @return {Promise<import('../../types').BlogData>}
  */
 const generateBlogData = async () => {
+  // Note: we cache the result of this function in a json file, we do this so that the cached
+  //       version can be read instead of re processing all the files, this is necessary for the
+  //       site to be deployed on a Cloudflare worker (which can't cope with reading hundred
+  //       of files)
+  const blogDataCacheFile = './.next/.nodejsorg-cache/blogData.json';
+  if (existsSync(blogDataCacheFile)) {
+    const rawContent = await readFile(blogDataCacheFile, 'utf8');
+    const content = JSON.parse(rawContent);
+    content.posts.forEach(p => (p.date = new Date(p.date)));
+    return content;
+  }
+
```

Comment:
@ovflowd 🥲

Reply:
Im discussing alternatives with @dario-piotrowicz on Slack DMs.

Reply:
I've implemented this differently (as recommended by @ovflowd 🙂)